packages/imperative/__tests__/__integration__/imperative/__tests__/__integration__/cli/config/convert-profiles/cli.imperative-test-cli.config.convert-profiles.integration.subtest.ts

```diff
@@ -142,17 +153,20 @@ describe("imperative-test-cli config convert-profiles", () => {
     describe("failure scenarios", () => {
         it("should not convert profiles if prompt is rejected", () => {
             const response = runCliScript(__dirname + "/__scripts__/convert_profiles.sh", TEST_ENVIRONMENT.workingDir, ["n"]);
-            expect(response.status).toBe(0);
+            expect(response.stderr.toString()).toEqual("");
             expect(response.stdout.toString()).toContain("Detected 2 old profile(s) to convert");
             expect(response.stdout.toString()).not.toContain("Your new profiles have been saved");
             expect(response.stdout.toString()).not.toContain("Your old profiles have been moved");
-            expect(response.stderr.toString()).toEqual("");
+            expect(response.status).toBe(0);
             expect(fs.existsSync(configJsonPath)).toBe(false);
         });
         it("should not delete profiles if prompt is rejected", () => {
-            runCliScript(__dirname + "/__scripts__/delete_profiles_secured_and_base.sh", TEST_ENVIRONMENT.workingDir);
+            // delete profiles previously created, but leave the profile type definitions
+            let response = runCliScript(__dirname + "/__scripts__/delete_profiles.sh", TEST_ENVIRONMENT.workingDir);
+            expect(response.stdout.toString()).toEqual("");
+            expect(response.stderr.toString()).toEqual("");
```

Comment:
Should we switch the order of these checks - 1st stderr and 2nd stdout? In case there is an error thrown, this makes it easier to debug tests because the error message logged to stderr will show up in the Jest output.

Reply:
That seems reasonable. I will switch them.